Makefile.core.mk

```diff
@@ -112,6 +112,13 @@ ifeq ($(USE_IMAGE_DIGESTS), true)
 	BUNDLE_GEN_FLAGS += --use-image-digests
 endif
 
+# Default flags used when rendering chart templates locally
+HELM_TEMPL_DEF_FLAGS = --include-crds --set image="${IMAGE}"
+# VALUES_FILE_OVERWRITE defines a values file to be used to overwrite default values from chart
+ifdef VALUES_FILE_OVERWRITE
```

Comment:
Maybe `VALUES_FILE_OVERRIDE` would be a slightly better name?

Reply:
actually just `VALUES_FILE` would be sufficient IMO

Reply:
yeah, a chart's default values are always taken into account even when supplying a values file or a set of custom values on the CLI. so I think it's clear enough